policy/runtime/runtime.go

```diff
@@ -704,15 +706,26 @@ func (pool *decisionSlotPool) Setup() decisionSlots {
 }
 
 type ruleActivation struct {
-	input     interpreter.Activation
-	rangeVars map[string]ref.Val
-	rule      model.Rule
+	input        interpreter.Activation
+	rangeVars    map[string]ref.Val
+	rule         model.Rule
+	tmplMetadata map[string]interface{}
+	instMetadata map[string]interface{}
 }
 
 func (ctx *ruleActivation) ResolveName(name string) (interface{}, bool) {
 	if name == "rule" {
 		return ctx.rule, true
 	}
+	if name == "template.properties" {
```

Comment:
I think the template.properties is actually just the template metadata exposed below. The difference between the `instance` and `template` is that the `instances` exposes a `map[string]string` and the template exposes a `struct` with well defined fields.

Reply:
hmmm.. I tried without this and the evaluation sth like `template.properties.env` did not work (with dummy properties values hardcoded)

Reply:
removed. 
as discussed offline, we will only support the "name" fields for now.